tests/tests.py

```diff
@@ -863,7 +863,7 @@ def test_calculate_offset_function(self):
         parent_joint_offset, child_joint_offset = modifier.calculate_offsets()
 
         self.assertEqual(parent_joint_offset, 0.5)
-        self.assertEqual(child_joint_offset, -0.5)
+        self.assertEqual(float(child_joint_offset[list(child_joint_offset.keys())[0]]), -0.5)
```

Comment:
The same comment as https://github.com/icub-tech-iit/urdf-modifiers/pull/24/files#r832063631 holds, in addition now child joint offset is a dict !

Reply:
I would advise against using raw dictionaries in general. 

Next time we need to change the code we will need to remember that the dictionary is supposed to contain (and under which keys) as it's not self-evident.

From what I understand the idea of the dict is to be able to have each offset and know to which joint it corresponds. Maybe we can create a small `Offset` class, that has two properties: the joint (or joint name if that's better) and the offset number itself.

That way `parent_joint_offset` can be an instance of the `Offset` class, and `child_joint_offset` can be an array of objects of this type. Then next time we look at the code we know exactly what the function returns and it's easier to maintain.

Of course we'd need to also change the functions that call this function to accept the new class and use it as intended.

What do you think @AlexAntn ?

Reply:
I agree a class would work better also on the long term, dicts tend to be hard to manage. What do you think @CarlottaSartore ?

Reply:
Changes implemented in [this](https://github.com/icub-tech-iit/urdf-modifiers/pull/24/commits/26668f313fef0775bd1801ae4440facab6093529) commit @CarlottaSartore @AlexAntn